wasm/store.go

```diff
@@ -716,43 +716,42 @@ func (s *Store) buildTableInstances(module *Module, target *ModuleInstance) (rol
 
 func (s *Store) buildExportInstances(module *Module, target *ModuleInstance) (rollbackFuncs []func(), err error) {
 	target.Exports = make(map[string]*ExportInstance, len(module.ExportSection))
-	for name, exp := range module.ExportSection {
```

Comment:
notice we don't really need the export section to be a map, and if it isn't, we retain insertion order, making late errors have more context